proto/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+//! Protobuf definitions for Penumbra.
+//!
+//! This crate only contains the `.proto` files and the Rust types generated
+//! from them.  These types only handle parsing the wire format; validation
+//! should be performed by converting them into an appropriate domain type, as
+//! in the following diagram:
+//!
+//! ```ascii
+//! ┌───────┐          ┌──────────────┐               ┌──────────────┐
+//! │encoded│ protobuf │penumbra_proto│ TryFrom/Into  │ domain types │
+//! │ bytes │<──wire ─>│    types     │<─validation ─>│(other crates)│
+//! └───────┘  format  └──────────────┘   boundary    └──────────────┘
+//! ```
```

Comment:
Eventually, we should have domain types that model the proto types, but for now (not sure for how long?), I think it's easier to just iterate on the proto design than on both protos + a set of domain types + conversions.